specification/maps/data-plane/Render/preview/2.1/render.json

```diff
@@ -0,0 +1,527 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Maps Render Service",
+    "version": "2.1",
+    "description": "Azure Maps Render REST APIs"
+  },
+  "host": "atlas.microsoft.com",
```

Comment:
Should we use "x-ms-parameterized-host" here? (Ref: 2.0)

Reply:
x-ms-parametrized-host should only be used in Creator, it will be removed across Azure Maps APIs in https://github.com/Azure/azure-rest-api-specs/pull/15386

I haven't touched 2.0 since it is addressed in that PR.